packages/slate-html-serializer/package.json

```diff
@@ -17,7 +17,7 @@
     "type-of": "^2.0.1"
   },
   "peerDependencies": {
-    "immutable": "^3.8.1",
+    "immutable": "^3.8.1 || ^4.0.0-rc",
```

Comment:
This is a good call @zhujinxuan. I think the ideal thing we should be doing is to instead make all of the peer dependencies use `>=` so that we never really have to update them like this. So this would change to:

```json
"immutable": ">=3.8.1",
```

And even the other ones simply:

```json
"react": ">=0.14.0",
"react-dom": ">=0.14.0",
...
```

Would you be down to make that change for all of them? I think it would end up making the codebase easier to manage for us and less restricting for everyone.